docs/faq-and-troubleshooting/common-questions/atlas-folder-missing.md

```diff
@@ -15,4 +15,4 @@ If you wish to have the folder on your desktop, you should right-click on the fo
 
 You can download the **.zip** archive below, extract it, and place the `AtlasDesktop` folder in `C:\Windows`. The link below updates upon each new Atlas release.
 
-[Download AtlasDesktop -->](https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2FAtlas-OS%2FAtlas%2Ftree%2F0.4.0%2Fsrc%2Fplaybook%2FExecutables%2FAtlasDesktop)
\ No newline at end of file
+[Download AtlasDesktop -->](https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2FAtlas-OS%2FAtlas%2Ftree%2Fmain%2Fsrc%2Fplaybook%2FExecutables%2FAtlasDesktop)
```

Comment:
This is intentional; the tag specified as `main` could be more updated than the current Playbook release